lib/flat_search/olx_scraper.ex

```diff
@@ -0,0 +1,116 @@
+defmodule OlxScraper do
+  use Crawly.Spider
+  alias FlatSearch.Flats
+
+  @impl Crawly.Spider
+  def base_url(), do: "https://www.olx.pl"
```

Comment:
do this need to be a function? maybe @base_url module variable?

Reply:
It's a template code form Crawly docs and it's default function but I thought the same. https://hexdocs.pm/crawly/readme.html

Reply:
ok 👍🏻